lib/appsignal/hooks/unicorn.rb

```diff
@@ -38,5 +39,13 @@ def close
         end
       end
     end
+
+    class UnicornProbe
+      def call
+        ObjectSpace.each_object(Unicorn::HttpServer) do |s|
+          Appsignal.set_gauge("unicorn_workers", s.worker_processes)
```

Comment:
```suggestion
          Appsignal.set_gauge("unicorn_worker_count", s.worker_processes)
```

What naming do we want to follow here? At least singular names as discussed before. So do we add `_count` to these kinds of metrics?

Reply:
I think so, we use `queue_length`, `worker_count` should be similar